utils/appliance.py

```diff
@@ -1076,6 +1077,8 @@ def restart_evm_service(self, log_callback=None):
                 msg = 'Failed to restart evmserverd on {}\nError: {}'.format(self.address, msg)
                 log_callback(msg)
                 raise ApplianceException(msg)
+        fire("server_config_changed")
```

Comment:
Is this the right place to put these? Does restarting the evm service imply that the server config has changed? perhaps these would be better placed in set_yaml_config?